src/renderer/components/message/Message.tsx

```diff
@@ -105,21 +105,26 @@ const ForwardedTitle = (
   const { displayName, color } = contact
 
   return (
-    <div
-      className='forwarded-indicator'
-      onClick={() => onContactClick(contact)}
-    >
-      {conversationType.hasMultipleParticipants && direction !== 'outgoing'
-        ? reactStringReplace(
-            tx('forwarded_by', '$$forwarder$$'),
-            '$$forwarder$$',
-            () => (
-              <span key='displayname' style={{ color: color }}>
-                {overrideSenderName ? `~${overrideSenderName}` : displayName}
-              </span>
-            )
+    <div className='forwarded-indicator'>
+      {conversationType.hasMultipleParticipants && direction !== 'outgoing' ? (
+        reactStringReplace(
+          tx('forwarded_by', '$$forwarder$$'),
+          '$$forwarder$$',
+          () => (
+            <span
+              onClick={() => onContactClick(contact)}
+              key='displayname'
+              style={{ color: color }}
+            >
+              {overrideSenderName ? `~${overrideSenderName}` : displayName}
```

Comment:
Little bit suprised about the reactStringReplace, can we not have the forwarded by string inside the span?

Reply:
other languages can have it at other places

Reply:
Ahh right! Gooot it